src/software/geom/util.h

```diff
@@ -522,7 +477,8 @@ std::pair<Point, Point> getCircleTangentPoints(const Point &start, const Circle
  * @param circle: The circle to calculate the tangent vectors of
  * @return the mean point of points
  */
-std::pair<Ray, Ray> getCircleTangentRays(const Point reference, const Circle circle);
+std::pair<Ray, Ray> getCircleTangentRaysToReference(const Point reference,
+                                                    const Circle circle);
```

Comment:
Not a huge fan of this name, but can't think of a better one off the top of my head. If anyone else has any suggestions, would appreciate it!

Reply:
getCircleTangentRaysWithReferenceOrigin() ?

Reply:
I changed the name because someone modified the function so that the origin of the rays WERE the tangent points on the circle.

I thought adding the 'reference' part made it more clear that the Rays were supposed to originate at the reference.

Reply:
I prefer `getCircleTangentRaysWithReferenceOrigin`. My main gripe here was that "reference" could really be anything, so specifying that it's a point (origin) resolves that.

Reply:
I still feel `ReferenceOrigin` isn't that much better, and is verbose for no reason. I don't think it provides more info / context that isn't already super obvious from the function signature. I find it more confusing than helpful, almost because it's trying to be _too_ helpful. The more verbose name wouldn't stop my from going to the function definition and reading the javadoc for a full explanantion.

Reply:
Renamed to Gareths suggestion

Reply:
At this point I'm not a huge fan of either name, but I don't have a better suggestion. @MathewMacDougall @EvanMorcom - if either of you have a strong opinion one way or the other, let's go with that. As far as I'm concerned this is resolved.

Reply:
Good with me :+1: